rc/filetype/elvish.kak

```diff
@@ -22,16 +22,17 @@ provide-module elvish %§
 add-highlighter shared/elvish regions
 add-highlighter shared/elvish/code default-region group
 add-highlighter shared/elvish/double_string region '"' (?<!\\)(\\\\)*" fill string
-add-highlighter shared/elvish/single_string region "'" (?<!')('')*' fill string
+add-highlighter shared/elvish/single_string region "'" ('')*' fill string
```

Comment:
The end pattern can actually be just `"'"` here. String literals that contain two single quotes (e.g. `'I''m'`) can just be highlighted as two separate string literals joined together.

Reply:
im pretty sure i made it like that because the simplest one was breaking on some edgecases. Cant check now.